docs/reference/deprecated.md

```diff
@@ -1,5 +1,9 @@
 Gutenberg's deprecation policy is intended to support backwards-compatibility for two minor releases, when possible. The current deprecations are listed below and are grouped by _the version at which they will be removed completely_. If your plugin depends on these behaviors, you must update to the recommended alternative before the noted version.
 
+## 3.7.0
+- `wp.editor.getColorName` has been removed. Please use `wp.editor.getColorObjectByColorValue` instead.
```

Comment:
It looks like we never use `wp.editor.getColorName` as public API in Gutenberg. What was the reason to make it public?

Would it make sense to keep only `getColorClass` public? By the way, should we rename `getColorClass` to `getColorClassName`?

Reply:
Now we don't use it because we use getColorObjectByColorValue. But it was being used in color-palette and panel-color.